Modules/Sources/DesignSystem/Gallery/ColorGallery.swift

```diff
@@ -89,7 +89,7 @@ struct ColorGallery: View {
     }
 
     private func hexString(for color: UIColor?) -> String? {
```

Comment:
I would remove this entire file. It's for a debug screen that are doubt we are going to need. The only color we are using from it, is ".brand" and it's best to render these in code like this:

<img width="388" alt="Screenshot 2024-08-06 at 3 06 52 PM" src="https://github.com/user-attachments/assets/a94ed6c8-38e6-496a-9ed6-0175685d13b9">

Reply:
Probably in a future PR, but yeah